MfciPkg/Docs/Mfci_Integration_Guide.md

```diff
@@ -33,15 +33,57 @@ Add the following to your EDK2 Flash Descriptor File (FDF)
 INF  MfciPkg/MfciDxe/MfciDxe.inf
 ```
 
+```INI
+INF  MfciPkg/MfciPei/MfciPei.inf
+```
+
 ### DSC
 
+#### Including Modules
+
 MfciPkg provides a ```.dsc.inc``` that can be ```!include``` in your platform DSC.
 An example follows:
 
 ```INI
 !include MfciPkg/MfciPkg.dsc.inc
 ```
 
+Mfci is still considered pre-production, so OPT_INTO_MFCI_PRE_PRODUCTION will need to be set at the top of the Dsc file

```

Comment:
Is this a merge conflict?  I thought we removed this text from 202202?  @kuqin12  @apop5

Reply:
Yes, I errantly added them back. 

I will put together a PR to drop them again.

Reply:
This should not be here.